packages/fxa-auth-server/lib/routes/subscriptions/stripe.ts

```diff
@@ -526,6 +546,16 @@ export class StripeHandler {
 
     const { paymentMethodId } = request.payload as Record<string, string>;
 
+    const paymentIntentCurrency = (
+      await this.stripeHelper.getPaymentIntent(paymentMethodId)
```

Comment:
Different object ids. The docs here describe how payment methods relate to payment intents:
https://stripe.com/docs/payments/payment-methods

Reply:
Thanks!